contracts/Contribution.sol

```diff
@@ -113,13 +108,39 @@ contract Contribution is Controlled, TokenController {
     Initialized(initializedBlock);
   }
 
-  function whitelist(address investor) {
+  /// @notice interface for founders to blacklist investors
+  /// @param _investors array of investors
+  function blacklistAddresses(address[] _investors) public onlyController {
+    for (uint256 i = 0; i < _investors.length; i++) {
+      address investorAddress = _investors[i];
```

Comment:
could we move this logic and call the `whitelist(address investor)` function?

Reply:
perhaps a modifier called `onlyControllerOrSelf` could be implemented here

Reply:
`onlyControllerOrSelf` what do you mean? 
1. I agree, I can call `whitelist` method